### PR TITLE
fix: minor fixes for context

### PR DESCRIPTION
**Pull request by wtrocki**
Closes # 

### Verification Steps

1. Do x
2. Do y

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation change
- [ ] Other (please specify)

### CODE REVIEW

File: pkg/core/localize/locales/en/cmd/context.en.toml [hunk 1]
```diff
@@ -127,7 +127,7 @@ one='Create a service context and assign associated service identifiers'
 [context.create.cmd.example]
 one='''
 ## Create context
-$ rhoas context create --name my-context
+$ rhoas context create --name dev
```
Comment (wtrocki) [inline pkg/core/localize/locales/en/cmd/context.en.toml:130]: Better reflecting what we do here

File: pkg/core/localize/locales/en/cmd/context.en.toml [hunk 2]
```diff
@@ -188,12 +188,14 @@ Use `rhoas context use-kafka` command to select a Kafka instance for your contex
 
 [context.common.error.registry.notFound]
 one='''
-Service registry instance doesn't exist, you need to set valid instance id using `rhoas context use-service-registry` command
+Service registry instance in your context doesnt exist.
+Your instance might have been removed outside rhoas-cli.
```
Comment (rkpattnaik780) [inline pkg/core/localize/locales/en/cmd/context.en.toml:192]: Not sure what it means.

**Review by rkpattnaik780 — APPROVED** [review]

**Comment by rkpattnaik780** [comment]
Not related to this PR, feel free to ignore.

Status command omits the name for current context.
```
./rhoas status                  
Name:
Location:       /home/rkpattnaik780/.config/rhoas/contexts.json

  Kafka
  --------------------------------------------------------------------------------
  ID:                     c90srhumk6hbcp2j04vg
  Name:                   hema-test
```

Can you add `opts.name = svcContext.CurrentContext` in this PR ? 
https://github.com/redhat-developer/app-services-cli/blob/fbb0c5a225e9860bbb5c17fb19de0a9d43e864b6/pkg/cmd/status/status.go#L101

File: pkg/core/localize/locales/en/cmd/context.en.toml (hunk 2 above)
Comment (wtrocki) [reply pkg/core/localize/locales/en/cmd/context.en.toml:192]: good point. Outside is bad word. Your instance have been removed sounds good overal

Comment (wtrocki) [reply pkg/core/localize/locales/en/cmd/context.en.toml:192]:
```suggestion
Your instance might have been removed.
```

**Comment by wtrocki** [comment]
Status command might need to be rewritten. See my suggestion on other PR

**Pull request merged**